lib/pluginmanager/update.rb

```diff
@@ -6,6 +6,7 @@ class LogStash::PluginManager::Update < LogStash::PluginManager::Command
   REJECTED_OPTIONS = [:path, :git, :github]
 
   parameter "[PLUGIN] ...", "Plugin name(s) to upgrade to latest version", :attribute_name => :plugins_arg
+  option "--[no-]verify", :flag, "verify plugin validity before installation", :default => true
```

Comment:
ok, then, ideally this should have been in another PR since this is not related to this PR intent.